021020.js

```diff
@@ -0,0 +1,24 @@
+let humanScore = 0;
+let computerScore = 0;
+let currentRoundNumber =  1;
+
+const generateTarget = () => {
+    return Math.floor(Math.random() * 10 );
+
+}
+
+const compareGuesses = (humanGuess, computerGuess, targetguess) =>
+{
+    const humanDifference = math.abs(targetGuess - humanGuess)

```

Comment:
Remember to capitalize the `Math` class name here or the `abs` method won't necessarily be referencing the right method.